.circleci/config.yml

```diff
@@ -128,22 +124,6 @@ jobs:
       - run: |
           kurtosis run ./simple-api/kurtosis-package/
 
-  check_simple_subpackage_example:
```

Comment:
I believe you also have to remove check_simple_subpackage_example in `workflows .build.jobs` below. That' likely why the CI is failing ATM